src/osdc/Objecter.cc

```diff
@@ -3166,6 +3181,30 @@ void Objecter::_finish_op(Op *op, int r)
   if (op->ontimeout && r != -ETIMEDOUT)
     timer.cancel_event(op->ontimeout);
 
+  ceph::coarse_mono_time now = ceph::coarse_mono_time::now();
+  auto latency = now - op->init_stamp;
+  int stat_type(0);
+  for (const auto& p : op->ops) {
+    switch (p.op.op) {
+      case CEPH_OSD_OP_READ:
```

Comment:
Nit: align `case` with `switch` indentation above

Reply:
I have modified it.Please review,thanks.